docs/sources/administration/plugin-management/_index.md

```diff
@@ -128,15 +139,15 @@ unzip my-plugin-0.2.0.zip -d YOUR_PLUGIN_DIR/my-plugin
 
 The path to the plugin directory is defined in the configuration file. For more information, refer to [Configuration](../../setup-grafana/configure-grafana/#plugins).
 
-#### Install a plugin using Grafana configuration
+### Install a plugin using Grafana configuration
 
 {{< admonition type="note" >}}
 This feature requires Grafana 11.5.0 or later.
 {{< /admonition >}}
 
 You can install plugins by adding the plugin ID to the `plugins.preinstall` section in the Grafana configuration file. This prevents the plugin from being accidentally uninstalled and can be auto-updated. For more information, refer to [Configuration](../../setup-grafana/configure-grafana/#plugins).
 
-#### Install a plugin in air-gapped environment
+### Install a plugin in air-gapped environment
 
 Plugin installation usually requires an internet connection. You can check which endpoints are used during the installation on your instance and add them to your instance’s allowlist.
```

Comment:
@tolzhabayev arguably this documents the endpoints (separate conversation relating to buckets)